File: examples/projects/benchmarks/loop_sum/loop_sum.go
// benchmark: 1 亿次 i64 循环累加 —— 与 loop_sum.zeta 同逻辑。输出 = 4999999950000000
package main

import "fmt"

func main() {
	var s int64
	for i := int64(0); i < 100000000; i++ {
		s += i
	}
	fmt.Println(s)
}

File: examples/projects/benchmarks/loop_sum/loop_sum.swift
// benchmark: 1 亿次 i64 循环累加 —— 与 loop_sum.rl 同逻辑
var s: Int64 = 0
var i: Int64 = 0
while i < 100000000 {
    s += i
    i += 1
}
print(s)
